driver.go

```diff
@@ -212,7 +212,10 @@ func (d *Driver) PreCreateCheck() error {
 
 	// Use a common key or create a machine specific one
 	if len(d.KeyPairName) != 0 {
-		d.SSHKeyPath = filepath.Join(d.StorePath, "sshkeys", d.KeyPairName)
+		key_path := filepath.Join(d.StorePath, "sshkeys", d.KeyPairName)
+		if _, err := os.Stat(key_path); err == nil {
+			d.SSHKeyPath = key_path
+		}
```

Comment:
Can you add a log line in a "else" case ? That will be helpful when troubleshooting connections issues? Maybe something like:

```
SSH key '/path/to/blah' does not exist. Assuming the key is in '~/.ssh/' or in a, AAS agent.
```